components/Content.js

```diff
@@ -3,19 +3,53 @@ import Link from 'components/Link'
 import parse, { domToReact } from 'html-react-parser'
 
 export default ({ content }) => (
-  <article className="prose max-w-none mb-8 last:mb-0 break-words">
+  <article className="max-w-none last:mb-0 break-words leading-7">
     {parse(content, parser)}
   </article>
 )
 
 const parser = {
```

Comment:
I don't think this parser thing is the right approach. See: https://github.com/trendwerk/nextjs-starter/issues/10#issuecomment-663188750.